src/handlers/fileResolver.js

```diff
@@ -1,11 +1,32 @@
 const path = require("path");
 const enhancedResolve = require("enhanced-resolve");
 const { defaultParsableExtensions } = require("../models/Parsables");
-
+const moduleResolver = require("babel-plugin-module-resolver");
+const findBabelConfig = require("find-babel-config");
 function checkNodeModules(absPath) {
-  return /node_modules/.test(absPath) || (absPath[0] !== "/" && absPath[1] !== ':');
+  return (
+    /node_modules/.test(absPath) || (absPath[0] !== "/" && absPath[1] !== ":")
+  );
 }
 
+function checkBabelModuleResolver(filePath, currentFile) {
+  const loadedConfig = findBabelConfig.sync(currentFile);
+  if (loadedConfig) {
```

Comment:
maybe also add some conditions if the config file does not contain plugins for example:
`loadedConfig && loadedConfig.config.plugins`